src/python/doc/datasets_generators.rst

```diff
@@ -0,0 +1,97 @@
+
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+===========================
+Datasets generators manual
+===========================
+
+We provide the generation of different customizable datasets to use as inputs for Gudhi complexes and data structures.
+
+
+Points generators
+------------------
+
+Points on sphere
+^^^^^^^^^^^^^^^^
+
+The module **_points** enables the generation of random i.i.d. points uniformly on a (d-1)-sphere in :math:`R^d`.
```

Comment:
Er, I really don't like a public name starting with `_`, especially when the same name without `_` is also public. You can for instance add the `import` you need in points.py so that we can use points.sphere without any underscore.

Reply:
Well, that's because two versions of torus are available. So we need to make the difference between the c++ wrapped version and the python one. If we choose to provide only the python implementation, we wouldn't need another name. Or do you suggest another option/name ?

Reply:
I imported `sphere` and `ctorus` (renamed cpp version of torus to avoid confusion) in points.

Reply:
> I imported `sphere` and `ctorus` (renamed cpp version of torus to avoid confusion) in points.

@mglisse this is my suggestion, we will have:
```python
from gudhi.datasets.generators import points

points.ctorus(n_samples = 64, dim = 3, sample = 'random') # c++ version
points.torus(n_samples = 64, dim = 3, sample = 'random') # python version
```
In order to keep points public.